server_admin/topics/users/user-registration.adoc

```diff
@@ -1,30 +1,16 @@
-[[_user-registration]]
+[id="assembly-user-registration_{context}"]
+= User Registration
 
-=== User Registration
+[role="_abstract"]
+You can use {project_name} as a third-party authorization server to manage application users. Administrators can add users or users can register themselves.
 
-You can enable {project_name} to allow user self registration.  When enabled, the login page has a registration
-link the user can click on to create their new account.
+You can enable {project_name} to allow user self-registration. When enabled, the login page displays a registration link that the user can click to create an account.
 
-When user self registration is enabled it is possible to use the registration form to detect valid usernames and emails.
```

Comment:
Joan, I am not sure how this statement about valid user names/emails works, but we should probably leave it in.

Reply:
@andymunro So, I actually checked this out. If a username is already taken you can not use it for a new registration. Similarly, if an email is provided that is not in the correct format,  it will also not be accepted (eg. joaedwar@redhat.com, not joaedwar-redhat.cm)

Reply:
I added the sentence back in. Please do not worry about these now... enjoy PTO :+1:

Reply:
Okay, now I get it.  It's probably a good call you made originally. Anyway thanks for explaining it.